src/bignum.h

```diff
@@ -560,31 +575,25 @@ inline const CBigNum operator%(const CBigNum& a, const CBigNum& b)
 {
     CAutoBN_CTX pctx;
     CBigNum r;
-    if (!BN_mod(&r, &a, &b, pctx))
+    if (!BN_div(r.get(), NULL, a.cget(), b.cget(), pctx))
         throw bignum_error("CBigNum::operator% : BN_div failed");
     return r;
 }
 
 inline const CBigNum operator<<(const CBigNum& a, unsigned int shift)
 {
     CBigNum r;
-    if (!BN_lshift(&r, &a, shift))
+    if (!BN_lshift(r.get(), a.cget(), shift))
         throw bignum_error("CBigNum:operator<< : BN_lshift failed");
     return r;
 }
 
-inline const CBigNum operator>>(const CBigNum& a, unsigned int shift)
```

Comment:
@ctya why do you remove CBugNum? See https://github.com/peercoin/peercoin/pull/153/commits/5b09830e5de0f5105534e69dbf4acffb3255869b line 598

Reply:
My mistake